<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MockTestQTClass</class>
 <widget class="QMainWindow" name="MockTestQTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MockTestQT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="DisplayManufacturerModel"/>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="Manufacturer">
        <property name="text">
         <string>Manufacturer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ManufacturerLine">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Model">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ModelLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="showAllButton">
        <property name="text">
         <string>Show All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="givenManufacturereButton">
        <property name="text">
         <string>Show given manufacturer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Show By Manufacturer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="manufacturerList"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MockTestQT.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>givenManufacturereButton</sender>
   <signal>clicked()</signal>
   <receiver>MockTestQTClass</receiver>
   <slot>showSortedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showSortedSlot()</slot>
 </slots>
</ui>
